pywinauto/uia_defines.py

```diff
@@ -42,7 +106,8 @@
 # header: UIAutomationClient.h
 
 def _build_pattern_ids_dic():
-    """A helper procedure to build a registry of control patterns 
+    """
```

Comment:
We need to agree about the style. PEP-8 sends to PEP-257 and the latter says that it can be trimmed to a single line. So I started fixing comments to be on the same line with the quotes and as I see you prefer to keep it separated.